packages/core/src/browser/xhrProxy.spec.ts

```diff
@@ -154,7 +159,29 @@ describe('xhr proxy', () => {
     })
   })
 
-  it('should track request with onreadystatechange overridden', (done) => {
+  it('should track request with onreadystatechange overridden before open', (done) => {
+    withXhr({
+      setup(xhr) {
+        xhr.onreadystatechange = () => undefined
```

Comment:
could it be interesting to ensure that this function is called?
(same for following test)